frb_dart/lib/src/basic.dart

```diff
@@ -88,11 +88,12 @@ abstract class FlutterRustBridgeBase<T extends FlutterRustBridgeWireBase> {
       if (isSuccess) {
         return task.parseSuccessData(rawReturn);
       } else {
-        throw FfiException('EXECUTE_SYNC', rawReturn as String, null);
+        if (rawReturn is String) {
+          throw FfiException('EXECUTE_SYNC', rawReturn, null);
+        } else {
+          throw task.parseErrorData!(rawReturn);
+        }
```

Comment:
Not sure whether the `String` case is indeed a panic? If so, maybe we have https://github.com/fzyzcjy/flutter_rust_bridge/blob/392a87f07d49e5df584633b5811a0eefa82da3ae/frb_dart/lib/src/basic.dart#L229

Reply:
refacted